src/Uno.UI/UI/Xaml/Media/Animation/Timeline.animation.Android.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Android.Animation;
+using Android.Views;
+using Android.Views.Animations;
+using Uno.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Windows.UI.Xaml.Media.Animation
+{
+	partial class Timeline
+	{
+		partial class AnimationImplementation<T>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['T' is not used in the class.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5301553)